docs/source/templates/taxonomy.md

```diff
@@ -83,6 +80,18 @@ The remote taxonomy must use JSON with items in the following format:
 | [`isLeaf`] | Boolean value. Use this instead of `children` when working with an API taxonomy. The default is `true`. When explicitly set to false, the node is treated as a parent node. [See below for more information](#API-taxonomies).  |
 | [`hint`] | This string will appear as a tooltip to the user when they hover their cursor over the value. |
 
+### Securing the taxonomy
+
+The `apiUrl` must be accessible to Label Studio. You can accomplish this in several ways:
+* The URL is public. 
+* The URL is hosted on a local server. 
+* The URL includes the username and password, for example `http://username:password@example.com/`. 
```

Comment:
could we add `The URL is a cloud storage URL pointing to a file within a source storage that's connected to the current project.`

Reply:
oh nevermind Markdown confused me, I see you covered this below :facepalm: